client/my-sites/themes/install-theme-button.jsx

```diff
@@ -83,8 +83,10 @@ const mapStateToProps = ( state ) => {
 };
 
 const mapDispatchToProps = ( dispatch ) => ( {
-	dispatchTracksEvent: ( { tracksEventProps } ) =>
-		dispatch( recordTracksEvent( 'calypso_click_theme_upload', tracksEventProps ) ),
+	dispatchTracksEvent: ( { tracksEventProps } ) => {
+		dispatch( recordTracksEvent( 'calypso_click_theme_upload', tracksEventProps ) );
+		dispatch( recordTracksEvent( 'calypso_themeshowcase_install_button_click', tracksEventProps ) );
```

Comment:
Two things of note here:
1) This isn't behind the feature flag `themes/showcase-i4/search-and-filter`. I'm fine with this, but open to suggestions 🙂
2) We already send `calypso_click_theme_upload`. I didn't remove it since it might break existing funnels.

Reply:
1. I'm fine with not putting Tracks events behind feature flags. They are cheap.
2. Cool!